nautobot/docs/configuration/authentication/sso.md

```diff
@@ -85,19 +96,19 @@ You will need to select the correct social authentication module name for your d
-| [Google](https://python-social-auth.readthedocs.io/en/latest/backends/google.html) | `social_core.backends.gae.GoogleAppEngineAuth` |
-| | `social_core.backends.google.GoogleOAuth2` |
-| | `social_core.backends.google.GoogleOAuth` |
-| | `social_core.backends.google_openidconnect.GoogleOpenIdConnect` |
-| [Okta](https://python-social-auth.readthedocs.io/en/latest/backends/okta.html) | `social_core.backends.okta.OktaOAuth2` |
-| | `social_core.backends.okta_openidconnect.OktaOpenIdConnect` |
-| [SAML](https://python-social-auth.readthedocs.io/en/latest/backends/saml.html) | `social_core.backends.saml.SAMLAuth` |
+| Backend                                                                                                       | Social Auth Backend Module Name                                 |
+| ------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------- |
+| [Microsoft Azure Active Directory](https://python-social-auth.readthedocs.io/en/latest/backends/azuread.html) | `social_core.backends.azuread.AzureADOAuth2`                    |
+|                                                                                                               | `social_core.backends.azuread_b2c.AzureADB2COAuth2`             |
+|                                                                                                               | `social_core.backends.azuread_tenant.AzureADTenantOAuth2`       |
+|                                                                                                               | `social_core.backends.azuread_tenant.AzureADV2TenantOAuth2`     |
+| [Google](https://python-social-auth.readthedocs.io/en/latest/backends/google.html)                            | `social_core.backends.gae.GoogleAppEngineAuth`                  |
+|                                                                                                               | `social_core.backends.google.GoogleOAuth2`                      |
+|                                                                                                               | `social_core.backends.google.GoogleOAuth`                       |
+|                                                                                                               | `social_core.backends.google_openidconnect.GoogleOpenIdConnect` |
+| [Okta](https://python-social-auth.readthedocs.io/en/latest/backends/okta.html)                                | `social_core.backends.okta.OktaOAuth2`                          |
+|                                                                                                               | `social_core.backends.okta_openidconnect.OktaOpenIdConnect`     |
+| [SAML](https://python-social-auth.readthedocs.io/en/latest/backends/saml.html)                                | `social_core.backends.saml.SAMLAuth`                            |
```

Comment:
I personally am not in favor of this change - it doesn't improve readability IMO.

Reply:
I will get this changed. It was my auto formatter x.x